src/plugins/discover/public/application/angular/doc_table/components/pager/tool_bar_pager_text.tsx

```diff
@@ -17,14 +17,12 @@ interface Props {
 
 export function ToolBarPagerText({ startItem, endItem, totalItems }: Props) {
   return (
-    <I18nProvider>
-      <div className="kuiToolBarText" data-test-subj="toolBarPagerText">
-        <FormattedMessage
-          id="discover.docTable.pagerControl.pagesCountLabel"
-          defaultMessage="{startItem}&ndash;{endItem} of {totalItems}"
-          values={{ startItem, endItem, totalItems }}
-        />
-      </div>
-    </I18nProvider>
+    <div className="kuiToolBarText" data-test-subj="toolBarPagerText">
+      <FormattedMessage
+        id="discover.docTable.pagerControl.pagesCountLabel"
+        defaultMessage="{startItem}&ndash;{endItem} of {totalItems} documents"
```

Comment:
Suppose, we should ping someone who can help with wording change

Reply:
Might be a little bit unclear seeing different numbers which refers to documents or pages. It was suggested [**here**](https://github.com/elastic/kibana/pull/103503#discussion_r666396983)

Reply:
@Dmitriynj  I think showing the total number of documents could be useful but not the "450-500 of" bit.  Also, could we add the "Rows per page" element at the bottom of the table? I think if we do that we could also remove the upper pagination and only have it at the bottom. How about something like this?

![image](https://user-images.githubusercontent.com/4016496/126240814-8c65ac81-d687-4bbc-b591-30622ebe73e2.png)

Reply:
As I understand the reason why we have two tool bars: at the top and bottom is that the user should permanently know total rows number and current page wether he/she on the top or bottom of the table. Well, we should probably make this two tool bars not scrollable, isn't it?

Reply:
Embeddable layout updated!